src/components/attendee/VideoHuddle/VideoHuddle.module.scss

```diff
@@ -8,31 +11,31 @@
   flex-direction: row-reverse;
 
   @media (min-width: 768px) {
-    @include sparkle-panel;
+    @include sparkle.panel--normal;
     grid-column: 1 / 3;
-    margin: 0 $whitespace * -1;
-    width: calc(100% + #{$whitespace * 2});
+    margin: 0 space.empty(-1);
+    width: calc(100% + #{space.empty(2)});
   }
 
   height: $video-chat-height;
-  padding: $button-gap;
-  border-top: 1px solid $sparkle-border;
-  border-bottom: 1px solid $sparkle-border;
+  padding: space.gap(1);
+  border-top: 1px solid theme.$sparkle--border;
+  border-bottom: 1px solid theme.$sparkle--border;
```

Comment:
maybe have something like `theme($sparkle--border)` instead?
I imagine we'll be in trouble if object nesting comes along